source/_integrations/lg_thinq.markdown

```diff
@@ -75,6 +76,39 @@ Support LG Appliances as follows:
 Air Purifier Fan, Washtower Dryer, Washtower Washer, Humidifier, Washcombo Main and Washcombo Mini are also supported.
 {% endnote %}
 
+## Platforms
+
```

Comment:
Nice! Thank you for adding those! 
- Instead of listing the technical property name, I think we could use the user-facing string. Please correct me if I got those suggestions wrong.
- Not sure if the resource columns is needed?
- Could you add a column that provides a short description of the functionality? For example, it is not clear to me what the Sabbath mode does or what the difference is between eco-friendly and power-saving?